ocs_ci/deployment/helpers/external_cluster_helpers.py

```
@@ -1651,11 +1693,49 @@ def get_and_apply_rgw_cert_ca(apply=True):
+                rgw_cert_ca_path,
+            )
+
+    # Older versions or version detection failed: Use rgw_cert_ca URL
     else:
+        if ceph_version is None:
+            logger.warning(
+                "Could not determine Ceph version, falling back to rgw_cert_ca URL"
+            )
+        else:
+            logger.info(
+                f"Ceph version {ceph_version} < 18.0, using rgw_cert_ca URL"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)